### ISSUE TITLE
Page Break Block Not Showing Pagination Links On Live Page

### ISSUE THREAD

**Issue by upwardmomentum84** [WordPress/gutenberg#54691]
### Description

When using Gutenberg 16.6.0 and you add a "Page Break" block to a page, the live page doesn't show the pagination links at the bottom of the page.

I tested GB 16.5.1 and other prior versions and the issue is not present.

### Step-by-step reproduction instructions

1. Make sure you are using a theme that uses the Site Editor
2. In a post or page, add a few paragraphs of text
3. Place a Page Break block in between a paragraph
4. Save the page
5. View the live version of the page

### Screenshots, screen recording, code snippet

https://github.com/WordPress/gutenberg/assets/93800099/b97c6a27-a0e6-4870-aa81-9cbe78291596

### Environment info

WP 6.3.1 and GB 16.6.0

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by jordesign** [comment]
I can confirm this issue when testing with GB 16.6 active. With Gutenberg plugin disabled it is not present.

**Comment by cuemarie** [comment]
This is still present for me in Gutenberg 16.8.1. Tested with no other plugins active and theme TT3 installed:

**Editor:**

![Markup on 2023-10-17 at 12:33:17](https://github.com/WordPress/gutenberg/assets/27249804/777d27ea-5858-4f96-90fd-016101a428fc)


**Live Site:**

![Markup on 2023-10-17 at 12:33:43](https://github.com/WordPress/gutenberg/assets/27249804/951c3b4a-5835-4bc5-b155-7b4a045ba4d3)

**Comment by annezazu** [comment]
Noting that I can't replicate for 6.4 RC1 so I will not be adding to the 6.4 board:

https://github.com/WordPress/gutenberg/assets/26996883/8377c138-39c1-4fc1-8113-7fe29b372ff7

I've added it to the Polish board as a quick, actionable fix and to raise some more awareness.

**Comment by annezazu** [comment]
Looping back on this as it was late last night and I was triaging for 6.4 to see if I can replicate with WP 6.3.2 and GB 16.8.1 to see if this issue is indeed replicable. 

I can replicate with WP 6.3.2 and GB 16.8.1.

Welcome other test reports to help narrow down but for now, doesn't seem like it's in 6.4 but it's narrowed down into the latest with GB.

**Comment by annezazu** [comment]
Looping back on this as it was late last night and I was triaging for 6.4 to see if I can replicate with WP 6.3.2 and GB 16.8.1 to see if this issue is indeed replicable.  I can replicate with WP 6.3.2 and GB 16.8.1.

Welcome other test reports to help narrow down but for now, doesn't seem like it's in 6.4 but it's narrowed down into the latest with GB.

**Comment by worldomonation** [comment]
I tried to reproduce this on various configurations, here are my findings:

**Gutenberg 16.8.1 + WP 6.3.3-nightly**
- nav links disappear (issue is reproduced)

**Gutenberg disabled + WP 6.3.3-nightly**
- nav links return (issue is no longer reproducible)

**Gutenberg 16.8.1 + WP 6.4-RC1**
- nav links are present (issue is not reproduced)

**Gutenberg disabled + WP 6.4-RC1**
- same as above (issue is not reproduced)

**Methodology**
Created a post with five paragraphs and inserted a Page Break block in between paragraphs 3 and 4. 
The post was then published.
After one variable was changed at a time, I returned to the published post and hard-refreshed the post.
